kustomizations/apps/data-hub-configs/kubernetes-pipeline--stg.config.yaml

```diff
@@ -1,46 +1,62 @@
+defaultConfig:
+  airflow:
+    dagParameters:
+      schedule: null
+      max_active_runs: 1
+      tags:
+        - 'Kubernetes'
+  env:
+    - name: DEPLOYMENT_ENV
+      value: '{ENV}'
+    - name: GOOGLE_APPLICATION_CREDENTIALS
+      value: /dag_secret_files/gcloud/credentials.json
+    - name: AWS_CONFIG_FILE
+      value: /dag_secret_files/aws/credentials
+  volumeMounts:
+    - name: gcloud-secret-volume
+      mountPath: /dag_secret_files/gcloud/
+      readOnly: true
+    - name: aws-secret-volume
+      mountPath: /dag_secret_files/aws
+      readOnly: true
+    - name: data-hub-config-volume
+      mountPath: /dag_config_files/
+      readOnly: true
+  volumes:
+    - name: aws-secret-volume
+      secret:
+        secretName: credentials
+    - name: gcloud-secret-volume
+      secret:
+        secretName: gcloud
+    - name: data-hub-config-volume
+      configMap:
+        name: data-hub-configs
+
 kubernetesPipelines:
 
   - dataPipelineId: 'Elife_Article_Xml_Pipeline_Kubernetes'
     airflow:
       dagParameters:
         schedule: '0 3 * * *'  # At 03:00, every day
-        max_active_runs: 1
-        tags:
-          - 'Kubernetes'
     image: '${data_hub_core_dags_unstable_image_repo}:${data_hub_core_dags_unstable_image_tag}'
     arguments:
       - 'python'
       - '-m'
       - 'data_pipeline.elife_article_xml.cli'
     env:
-      - name: DEPLOYMENT_ENV
-        value: '{ENV}'
       - name: ELIFE_ARTICLE_XML_CONFIG_FILE_PATH
         value: /dag_config_files/elife-article-xml.config.yaml
       - name: GITHUB_API_AUTHORIZATION_FILE_PATH
         value: /dag_secret_files/github_api/github_api_authorization.txt
-      - name: GOOGLE_APPLICATION_CREDENTIALS
-        value: /dag_secret_files/gcloud/credentials.json
     volumeMounts:
-      - name: github-api-secret-volume
-        mountPath: /dag_secret_files/github_api/
-        readOnly: true
       - name: gcloud-secret-volume
```

Comment:
Should we have kept `github-api-secret-volume` instead?

Reply:
Ah yes, I was trying to find the reason of the issue. Thanks!